index.html

```diff
@@ -4,16 +4,17 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <title>Document</title>
+    <meta name="description" content="Shayan portfolio website convering shayan cv and resume"> 
+    <title>Portfolio</title>
```

Comment:
- [ ] Great job with building a well-accessible portfolio page 👍 👍 but this title is not very descriptive as it is too generic. Kindly make changes to this `title` content to reflect the purpose of your page. How about using something like `Shayan's Portfolio`? 😉